homeassistant/components/apcupsd/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 from .coordinator import APCUPSdConfigEntry, APCUPSdCoordinator
 
-PLATFORMS: Final = (Platform.BINARY_SENSOR, Platform.SENSOR)
+PLATFORMS: Final[list[Platform]] = [Platform.BINARY_SENSOR, Platform.SENSOR]
```

Comment:
The type is already inferred.

```suggestion
PLATFORMS: Final = [Platform.BINARY_SENSOR, Platform.SENSOR]
```

Reply:
👍 committed the change here, thanks!